forbidigo/forbidigo.go

```diff
@@ -164,14 +168,54 @@ func (v *visitor) Visit(node ast.Node) ast.Visitor {
 		}
 		return v
 	case *ast.SelectorExpr:
+		selectorExpr = node
```

Comment:
If we pull out the package name expander, I think we could stop collecting this here and just re-match against `*ast.SelectorExpr` in the helper method I proposed.